docs/data/data-grid/column-definition/RenderCellGrid.tsx

```diff
@@ -1,25 +1,55 @@
 import * as React from 'react';
 import Button from '@mui/material/Button';
+import { TouchRippleActions } from '@mui/material/ButtonBase/TouchRipple';
```

Comment:
This is a private import, it's not allowed 🚫, we will need to change this. More details in https://mui.com/material-ui/guides/minimizing-bundle-size/